src/components/avatar/Avatar.tsx

```diff
@@ -19,21 +21,29 @@ export interface AvatarProps {
     className?: string;
     /** Users id */
     id?: string | number | null;
+    /** Whether this avatar should be labeled as external in the current context */
+    isExternal?: boolean;
     /**
      * Users full name.
      *
      * Required if "avatarUrl" is not specified.
      */
     name?: string | null;
+    /** Show the external avatar marker if the avatar is marked as for an external user */
+    shouldShowExternal?: boolean;
     /* avatar size (enum) */
     size?: keyof typeof SIZES | '';
 }
 
-function Avatar({ avatarUrl, className, name, id, size = '' }: AvatarProps) {
+function Avatar({ avatarUrl, className, name, id, isExternal, shouldShowExternal = false, size = '' }: AvatarProps) {
     const [hasImageErrored, setHasImageErrored] = React.useState<boolean>(false);
     const [prevAvatarUrl, setPrevAvatarUrl] = React.useState<AvatarProps['avatarUrl']>(null);
 
-    const classes = classNames(['avatar', className, { [`avatar--${size}`]: size && SIZES[size] }]);
+    const classes = classNames([
+        'avatar',
+        className,
+        { [`avatar--${size}`]: size && SIZES[size], 'avatar--isExternal': shouldShowExternal && isExternal },
```

Comment:
Should these classes be namespace'd and SUIT convention'ed?

Reply:
So s/`avatar`/`bdl-Avatar`/ ?